src/main/java/com/heeverse/config/VaultEnvironmentConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.heeverse.config;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.vault.authentication.ClientAuthentication;
+import org.springframework.vault.authentication.TokenAuthentication;
+import org.springframework.vault.client.VaultEndpoint;
+import org.springframework.vault.config.AbstractVaultConfiguration;
+
+/**
+ * @author jeongheekim
+ * @date 2023/08/03
+ */
+@Configuration
+public class VaultEnvironmentConfig extends AbstractVaultConfiguration {
+
+    @Value("${vault.scheme}")
+    private String scheme;
+
+    @Value("${vault.uri}")
+    private String url;
+
+    @Value("${vault.token}")
+    private String token;
+
+    @Value("${vault.port}")
+    private int port;
+
+    @Override
+    public VaultEndpoint vaultEndpoint() {
+        String authority = url + ":" + port;
+        URI uri = null;
+        try {
+            uri = new URI(scheme, authority, null, null, null);
+        } catch (URISyntaxException e) {
+            throw new RuntimeException(e);
+        }
+        return VaultEndpoint.from(uri);
+    }
```

Comment:
이렇게 바꾸면 catch문 이후를 안봐도 된다고 표현할 수 있으니, 읽는사람의 가독성을 향상시켜줄 수 있습니다. 

```suggestion
    @Override
    public VaultEndpoint vaultEndpoint() {
        String authority = url + ":" + port;
        try {
            return VaultEndpoint.from(new URI(scheme, authority, null, null, null));
        } catch (URISyntaxException e) {
            throw new RuntimeException(e);
        }   
    }
```